src/libpriv/rpmostree-postprocess.c

```diff
@@ -1559,6 +1556,10 @@ rpmostree_treefile_postprocessing (int            rootfs_fd,
       }
   }
 
+  /* Copy in additional files before postprocessing */
+  if (!copy_additional_files (rootfs_fd, context_directory, treefile, cancellable, error))
```

Comment:
I just realized now that we handle `add-files` *after* the `/etc` to `/usr/etc` move. What this patch does is correct since it maintains that, though are we sure that's what we want to begin with? Seems like users would expect to dump things in e.g. `/etc`, not `/usr/etc`, right?

Reply:
FWIW, I'd prefer the <code>/etc</code> to <code>/usr/etc</code> move happen after both <code>add-files</code> and the <code>post</code> script, but it's easy enough to work around if that's not possible.

Reply:
I'm not totally opposed to changing it but on the other hand we had a test case that did it, and I've definitely seen people talk about using `add-files` here and I'd like not to break them.

Perhaps the simplest thing is to support *both*; we could do that by dropping a temporary symlink, or translating paths.

Reply:
Translating paths seems reasonable, yeah.